app/services/bookings_services.rb

```diff
@@ -0,0 +1,31 @@
+class BookingsServices
```

Comment:
**Code Critical - Refactor**

Introduce the `BookingsServices` service object to refactor booking logic.